nipoppy/workflow/proc_pipe/tractoflow/run_tractoflow.py

```diff
@@ -21,22 +22,59 @@
 
 MEM_MB = 4000
 
-def parse_data(bids_dir, participant_id, session_id, logger=None):
+def parse_data(global_configs, bids_dir, participant_id, session_id, use_bids_filter=False, logger=None):
     """ Parse and verify the input files to build TractoFlow's simplified input to avoid their custom BIDS filter
     """
 
+    ## load from global configs
+    DATASET_ROOT = global_configs["DATASET_ROOT"]
+
     ## because why parse subject ID the same as bids ID?
     subj = participant_id.replace('sub-', '')
```

Comment:
Not urgent, but there are helper functions and global variables now for different ID conversions in [utils.py](https://github.com/neurodatascience/nipoppy/blob/main/nipoppy/workflow/utils.py)